modules/apps/foundation/frontend-theme/frontend-theme-user-dashboard/src/templates/portal_normal.ftl

```diff
@@ -52,15 +52,11 @@
 					</#if>
 				</div>
 
-				<#assign VOID = freeMarkerPortletPreferences.setValue("portletSetupPortletDecoratorId", "barebone") />
```

Comment:
Hello!

If we remove these lines the design will be different than originally was designer, so.. If this is a real and big performance problem, please let me know and we will try to get the design in other way.

Thanks a lot!